src/main/java/uk/gov/hmcts/ccd/endpoint/std/CaseSearchEndpoint.java

```diff
@@ -67,45 +59,25 @@ public CaseSearchResult searchCases(
             + "the search results will contain only metadata by default (no case field data). To get case data in the "
             + "search results, please state the alias fields to be returned in the _source property for e.g."
             + " \"_source\":[\"alias.customer\",\"alias.postcode\"]",
-                  required = true)
+            required = true)
         @RequestBody String jsonSearchRequest) {
 
         Instant start = Instant.now();
 
-        rejectBlackListedQuery(jsonSearchRequest);
+        CrossCaseTypeSearchRequest request = elasticsearchQueryHelper.prepareRequest(
+            caseTypeIds,
+            UseCase.DEFAULT.getReference(),
+            jsonSearchRequest
+        );
 
-        CrossCaseTypeSearchRequest request = new CrossCaseTypeSearchRequest.Builder()
-            .withCaseTypes(caseTypeIds)
-            .withSearchRequest(stringToJsonNode(jsonSearchRequest))
-            .build();
-
-        CaseSearchResult result = caseSearchOperation.execute(request);
+        CaseSearchResult result = caseSearchOperation.executeExternal(request);
```

Comment:
in the code base methods we don't really mention external vs internal. I'd keep it agnostic as simply `execute`.